src/release/breaking-changes/toggleable-active-color.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ThemeData's toggleableActiveColor property has been deprecated
+description: Material Widgets that use toggleableActiveColor property are migrated to use Material ColorScheme.
+---
+
+## Summary
+
+The Material widgets `Switch`, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` now use `ColorScheme.secondary` color for their toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
+
+## Context
+
+Migration of widgets that depends on `ThemeData.toggleableActiveColor` to `ColorScheme.secondary` rendered toggleableActiveColor unuse, hence the deprecationn of this theme property.
+
+## Description of change
+
+The widgets using `ThemeData.toggleableActiveColor` color for the active/selected state now use `ColorScheme.secondary`.
+
+## Migration guide
+
+Toggleable widgets' active/selected color can generally customized in 3 ways at the widget-level using components themes `SwitchThemeData`, `ListTileThemeData`, `CheckboxThemeData`, and `RadioThemeData` using `ColorScheme.secondary`.
```

Comment:
@TahaTesser, I'm struggling with the following sentence. What are the 3 ways
you can customize the toggleable widget's active color? You list 4 ThemeData
classes and the ColorScheme class. So, I count either 5 ways, or 2 ways,
depending on how you count. Not 3. Can you clarify?
```suggestion
Toggleable widgets' active/selected color can generally customized in 3 ways
at the widget level using components themes `SwitchThemeData`, `ListTileThemeData`,
`CheckboxThemeData`, and `RadioThemeData` using `ColorScheme.secondary`.
```

Reply:
Thanks so much, I realized that it wasn't clear. I just updated this section.